.github/workflows/build.yml

```diff
@@ -837,7 +881,24 @@ jobs:
           # On develop branch, we want to _store_ the cache (so it can be used by other branches), but never _restore_ from it
           restore-keys: ${{ env.NX_CACHE_RESTORE_KEYS }}
       - name: Build tarballs
-        run: yarn build:tarball
+        run: yarn build:tarball --ignore @sentry/profiling-node
+
+      # Rebuild profiling by compiling TS and pulling the precompiled binaries
+      - name: Build Profiling Node
+        run: yarn lerna run build:lib --scope @sentry/profiling-node
+
+      - name: Extract Profiling Node Prebuilt Binaries
+        # @TODO: v4 breaks convenient merging of same name artifacts
+        # https://github.com/actions/upload-artifact/issues/478
+        uses: actions/download-artifact@v3
```

Comment:
Can we use the workaround in https://github.com/actions/upload-artifact/issues/478#issuecomment-1885470013 here?